pyscriptjs/tests/integration/test_async.py

```diff
@@ -148,6 +151,7 @@ async def a_func():
             == "Implicit target not allowed here. Please use display(..., target=...)"
         )
 
+    @pytest.mark.xfail(reason="fails after introducing synclink, fix me soon!")
```

Comment:
writing this comment as a reminder: we decided to fix these tests in a follow-up PR